### PR TITLE
flamenco: gossip version set, no more tmp contact info

### PR DESCRIPTION
**Pull request by lheeger-jump**

### CODE REVIEW

File: src/flamenco/gossip/fd_gossip.c [hunk 1]
```diff
@@ -853,7 +805,7 @@ fd_gossip_random_pull( fd_gossip_t * glob, fd_pending_event_arg_t * arg ) {
   (void)arg;
 
   /* Try again in 5 sec */
-  fd_pending_event_t * ev = fd_gossip_add_pending(glob, glob->now + (long)5e9);
+  fd_pending_event_t * ev = fd_gossip_add_pending(glob, glob->now + (long)100e6);
```
Comment (arjain4) [inline src/flamenco/gossip/fd_gossip.c:808]: what is this change all about?

**Review by arjain4 — APPROVED** [review]
looks good

File: src/flamenco/gossip/fd_gossip.c (hunk 1 above)
Comment (lheeger-jump) [reply src/flamenco/gossip/fd_gossip.c:808]: it was too slow!

Comment (lheeger-jump) [reply src/flamenco/gossip/fd_gossip.c:808]: we will adjust it

**Pull request merged**